packages/suite/src/components/suite/Labeling/components/Metadata/index.tsx

```diff
@@ -13,25 +13,25 @@ import { withDropdown } from './withDropdown';
 
 import type { Timeout } from '@trezor/type-utils';
 
-const LabelDefaultValue = styled.div`
-    width: 0;
-    display: inline-block;
-    transition: width 0.6s, visibility 0.3s, opacity 0.3s;
-    visibility: hidden;
+const LabelValue = styled.div`
+    overflow: hidden;
+    text-overflow: ellipsis;
+`;
+
+const LabelDefaultValue = styled(LabelValue)`
+    /* do not shrink when the expanded label does not fit the contener - shrink only the label value */
+    flex-shrink: 0;
+    max-width: 0;
+    /* transition max-width because it does not work with auto value */
+    transition: max-width 0.6s, opacity 0.3s;
```

Comment:
nit: `0.6s` It seems slow to me

Reply:
b406e14399ceff4177e5ab91e78e4e775ad2fd70 increased speed to 0.3s.

Reply:
I usually make all micro-interactioncs in the 0.2 - 0.25 range

Reply:
Further increased to 0.25s in bbb51e3f3e834bf8f2b63a5e93c20ab65061c502.